projects/forge-lib/src/lib/modules/form-builder/components/field-editor/field-editor.component.ts

```diff
@@ -221,6 +221,34 @@ export class FieldEditorComponent implements OnInit {
         this.showAbstractDisplay = true;
         this.showAbstractData = true;
         this.showAbstractValidation = true;
+        break;
+      }
+      case ComponentTypes.Currency: {
```

Comment:
Does the the code coverage meet the 80% rule here with updates?

Reply:
Now it should be > 80% coverage, after my most recent push.